core/node/shared/ids.go

```diff
@@ -123,7 +123,7 @@ func UserSettingStreamIdFromAddr(addr common.Address) StreamId {
 	return b
 }
 
-func UserDeviceKeyStreamIdFromAddress(addr common.Address) StreamId {
+func UserPublicDataStreamIdFromAddress(addr common.Address) StreamId {
 	var b StreamId
 	b[0] = STREAM_USER_DEVICE_KEY_BIN
```

Comment:
Should we not also rename `STREAM_USER_DEVICE_KEY_BIN`?

Reply:
yes good catch